packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -30,6 +30,59 @@ import 'tooltip_theme.dart';
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=EeEfD5fI-5Q}
 ///
+/// ## Example
+///
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// This example show a basic [Tooltip] which has a [Text] as child.
+/// [message] contains your label to be shown on long pressed.
```

Comment:
nit: slight suggestion to clarify what exactly is being long pressed:
```suggestion
/// [message] contains your label to be shown by the tooltip when
/// the child that Tooltip wraps is long pressed.
```